aiohttp/server.py

```diff
@@ -90,8 +89,7 @@ def __init__(self, *, loop=None,
                  keep_alive_on=True,
                  timeout=0,
                  logger=server_logger,
-                 access_log=None,
```

Comment:
`'%(i_FOOBAR)s'` looks ugly, I really like `%{FOOBAR}i`.
 I've got your point.

What's about adding `access_logger` parameter but keeping `access_log` and `access_log_format` for a while?

If `access_logger` is None construct logger from `access_log` and `access_log_format`, use `access_logger` otherwise. We are keeping backward compatibility for old users but adding new functionality.

Make sure `access_logger` and (`access_log`, `access_log_format`) pair are mutually exclusive. Raise ValueError if both are specified.

Feel free to making `access_log` and `access_log_format` parameters deprecated, we can get rid of these after deprecation period.